sos/report/plugins/openvswitch.py

```diff
@@ -109,6 +98,18 @@ def setup(self):
         self.add_journal(units="ovs-vswitchd")
         self.add_journal(units="ovsdb-server")
 
+        # Gather the datapath information for each datapath
+        dp_list_result = self.collect_cmd_output('ovs-appctl dpctl/dump-dps')
+        if dp_list_result['status'] == 0:
+            for dp in dp_list_result['output'].splitlines():
+                self.add_cmd_output([
+                    "ovs-appctl dpctl/show -s %s" % dp,
```

Comment:
would be good to add a comment in commit msg on why changing from ovs-dpctl to ovs-appctl

Reply:
will do